storage/txcache/testutils_test.go

```diff
@@ -0,0 +1,90 @@
+package txcache
```

Comment:
There are utility functions shared among more test files. Since they are used by more tests (separate files), could not decide where to move them :)

We can use this pattern in the future. Shared test utilities could stay in such a file.

Reply:
can remain like this although we avoided using "utils" or "helpers"